### PR TITLE
Add migration documentation

### PR DESCRIPTION
**Pull request by veolys-nanne**

### CODE REVIEW

File: en/hosting_guide/migration.rst [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+dir_name: as the use of this bundles will result in generation of files, the target is specified
+script_dir_name: same as dir_name, but this indicates where js script are located
+name: arbitrary value used in output
+namespace: this value should be consistent with dir_name
+
+MongoDb Migration generate
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The command, app/console mongodb:migrations:generate, generates a blank file in dir_name.
+This file is characterized by a unique identifier, stored in database, which indicates the order of the file in the migration process.
```
Comment (nicolasThal) [inline en/hosting_guide/migration.rst:44]: could you split the lines so there is no line return in the code ?

**Comment by alavieille** [comment]
can you add this documentation in summary.

**Comment by veolys-nanne** [comment]
@nicolasThal, @itkg-alavieille done

File: en/hosting_guide/migration.rst [hunk 2]
```diff
@@ -0,0 +1,67 @@
+Migration
+=========
+
+To know more about migration see `mongodb migration`_.
+
+MongoDB Migration
+-----------------
+
+After having install MongoDBMigrationsBundle, it is possible to list the different allowed commands with :
```
Comment (nicolasThal) [inline en/hosting_guide/migration.rst]: You should rephrase it with : the MongoDBMigrationsBundle come with a set of command available under the name space `mongodb:migrations`

**Comment by veolys-nanne** [comment]
done @nicolasThal , @itkg-alavieille

**Comment by alavieille** [comment]
:+1:

**Comment by nicolasThal** [comment]
:+1:

**Pull request merged**